tiny-ab.js

```diff
@@ -0,0 +1,41 @@
+const tinyABinit = () => {
+  const contentOriginal = document.getElementsByClassName('ab-test-original');
+  const contentAlt = document.getElementsByClassName('ab-test-alternative');
+  const urlParams = new URLSearchParams(window.location.search);
+  const testCriteria = (contentAlt.length > 0) && (urlParams.has('lti_context_id'));
+
+  if (testCriteria) {
+    const id = urlParams.get('lti_context_id');
+    addCSS();
+    runTinyAB(contentOriginal, contentAlt, id);
+  }
+}
+
+const addCSS = () => {
+  const style = document.createElement('style');
+  style.innerHTML = `
+    .ab-test-original+.ab-test-alternative { display: none; }
+  `;
+  document.head.appendChild(style);
+}
+
+const runTinyAB = (contentOriginal, contentAlt, id) => {
+  const lastChar = id.slice(-1);
+  const altChars = ['1', '3', '5', '7', '9', 'b', 'd', 'f'];
+  const showAlt = altChars.includes(lastChar);
+
```

Comment:
I thought about this, too, but wanted to stick to the specifics outlined by David (specifically around B, D, F) since I don't know what other possibilities/options exist in LTI Context IDs.